setup.py

```diff
@@ -2,6 +2,7 @@
 from setuptools import setup, find_packages, Extension
 from setuptools.command.build_ext import build_ext as _build_ext
 import warnings
+import numpy #could not find numpy headers fix
```

Comment:
Numpy might not be available. fastdtw is (supposed to be?) installable in environments without a compiler, defaulting to slow Python implementation.
Can you make the import more conditional?